packages/web3-core-method/src/index.js

```diff
@@ -819,6 +815,112 @@ Method.prototype.buildCall = function () {
     return send;
 };
 
+function _handleTxType(tx) {
```

Comment:
This is duplicate code from web3-eth-accounts.

Why don't we push these two functions to web3-eth-helpers, since both web3-core-method and web3-eth-accounts import it?

Reply:
We should definitely do this if there aren't any crazy refactors involved. I think adding it to `web3-eth-helpers` might solve the need for a provider, but idk if `web3-core-method` needing to passing in `method` [here](https://github.com/ChainSafe/web3.js/blob/b5dadf9ef2352a1378c26937d03d4109fd3319b0/packages/web3-core-method/src/index.js#L781) and `web3-eth-accounts` needing to pass in `_this` [here](https://github.com/ChainSafe/web3.js/blob/b5dadf9ef2352a1378c26937d03d4109fd3319b0/packages/web3-eth-accounts/src/index.js#L270) are going to be an issue. It looks like in both instances, `method` and `_this` are just used to make the RPC call, but they seem to go about it differently